.github/workflows/release-ruby.yml

```diff
@@ -21,11 +21,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - name: Synchronize CI Dict
-        working-directory: ruby
-        run: bundle exec rake copy_ci_dict
-
```

Comment:
I am not sure that step can be removed without risk

Reply:
Ok, fair point. I suggest:

- Restore `run: bundle exec rake lib/cucumber/ci_environment/CiEnvironments.json`
- Add a step at the end of the CI build that fails the build if there are dirty files in the repo.

Reply:
Agree